roles/poa-netstats/tasks/main.yml

```diff
@@ -32,7 +32,6 @@
 - name: install npm netstats
   npm:
     path: "{{ home }}/eth-net-intelligence-api"
-  become: true
```

Comment:
is this a typo? In other places we have both `become` and `become_user`

Reply:
Looks like. Fixed.